breadth_first_search/BreadhFirstSearch.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <list>
+using namespace std;
+
+class Graph {
+	int v;
+	list<int> *adjlist;
+	public :
+
+	Graph(int V) {
```

Comment:
Capital letters are reserved for constants, avoid using all caps for local variables.